ui_testing/testcases/basic_tests/test06_orders.py

```diff
@@ -2296,3 +2300,40 @@ def test065_download_suborder_sheet_for_single_order(self):
                                   f"{str(fixed_sheet_row_data)} : {str(formatted_orders[index])}")
             for item in formatted_orders[index]:
                 self.assertIn(item, fixed_sheet_row_data)
+
+    @parameterized.expand(['sub_script', 'super_script'])
+    def test068_filter_testunit_by_scripts(self, value):
+        """  Make sure that user can filter by sub & super scripts in the filter drop down list
+             LIMS-7447
+        """
+
+        self.test_unit_api.set_name_configuration()
+        if value == 'sub_script':
+            api, testunit = self.test_unit_api.create_qualitative_testunit(unit='14[158]')
+            self.assertEqual(api['status'], 1)
+        else:
+            api, testunit = self.test_unit_api.create_qualitative_testunit(unit='15{158}')
+            self.assertEqual(api['status'], 1)
+
+        unit = testunit['unit']
+        self.orders_page.get_orders_page()
+        self.order_page.sleep_small()
+        self.base_selenium.scroll(False)
+        self.orders_page.apply_filter_scenario(filter_element='orders:test_units_filter',
+                                               filter_text=unit, field_type='drop_down')
+        self.assertIsNotNone(self.base_selenium.is_text_included_in_drop_down_items(element='orders:test_units_filter', item_text=unit))
```

Comment:
in Jira ticket of this test case, step 5:
you need to assert that result display.
So you will create order with this test unit then get filter result data and assert that unit found in it